doc/source/index.rst

```diff
@@ -211,6 +211,7 @@ Papers
    iter.rst
    pandas_on_ray.rst
    dask-on-ray.rst
+   mars_on_ray.rst
```

Comment:
can you use dashes instead of underscores?

Reply:
Fixed